imgur_uploader.py

```diff
@@ -55,18 +55,19 @@ def upload_image(image):
         client = ImgurClient(config["id"], config["secret"])
         anon = True
 
-    click.echo("Uploading file {}".format(click.format_filename(image)))
+    for image in images:
+        click.echo("Uploading file {}".format(click.format_filename(image)))
 
-    response = client.upload_from_path(image, anon=anon)
+        response = client.upload_from_path(image, anon=anon)
 
-    click.echo("File uploaded - see your image at {}".format(response["link"]))
+        click.echo("File uploaded - see your image at {}".format(response["link"]))
 
-    try:
-        import pyperclip
+        try:
+            import pyperclip
 
-        pyperclip.copy(response["link"])
-    except ImportError:
-        print("pyperclip not found. To enable clipboard functionality," " please install it.")
+            pyperclip.copy(response["link"])
```

Comment:
This will only copy the last image URL to the clipboard.

Reply:
yep, but it will have those elements in your clipboard history which makes it easy to paste previous links.

Reply:
Alternatively, it is possible to concatenate all links to one entry but I felt the clipboard solution was neater as I could reference the links exactly where I want them since they're separate. I'm happy to modify it as you want, @skorokithakis 

![image](https://user-images.githubusercontent.com/27063113/95668466-f1cf7300-0b91-11eb-810c-315268f708c7.png)

Reply:
Hm, yes, the issue with this is that the clipboard history thing only works on certain OSes. I think it would be better if the plugin copied a list of URLs in the end, one per line, what do you think?

Reply:
> Hm, yes, the issue with this is that the clipboard history thing only works on certain OSes. I think it would be better if the plugin copied a list of URLs in the end, one per line, what do you think?

What if we did both? It adds the individual links as they get uploaded and then finally appends a list of them to the clipboard. That way, people with clipboard managers can still reference individual links as they need and people without clipboard history will not lose access to all the links except the last one.

Reply:
Yep, that sounds good!

Reply:
done!

Reply:
Looks good! Can you add a comment explaining why we copy both each line and all of them at the end? I can see someone getting confused if they just read the code later without a comment to explain.

Reply:
good idea! I added comments